### PR TITLE
owners: add puerco/cpanato to the sig-release-leads

### PR DESCRIPTION
**Pull request by cpanato**
#### What type of PR is this:
/kind documentation

#### What this PR does / why we need it:

As part of the onboard described in this issue: https://github.com/kubernetes/sig-release/issues/1590

#### Special notes for your reviewer:

/assign @justaugustus @saschagrunert @jeremyrickard @hasheddan

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -1,10 +1,12 @@
 aliases:
   sig-release-leads:
     - alejandrox1 # SIG Technical Lead
```
Comment (justaugustus) [inline OWNERS_ALIASES:3]:
```suggestion
```

**Review by justaugustus — CHANGES_REQUESTED** [review]
@cpanato -- Let's cleanup and remove Jorge as part of this PR too.

**Review by justaugustus — COMMENTED** [review]
Also, let's update SECURITY_CONTACTS as part of the PR.

File: OWNERS_ALIASES (hunk 1 above)
Comment (cpanato) [reply OWNERS_ALIASES]: done

**Review by justaugustus — APPROVED** [review]
Weeeeeeee! 🎉

**Pull request merged**